tests/manager/api/test_axis.py

```diff
@@ -774,6 +777,8 @@ def test_update_book(self, axis360: Axis360Fixture):
             licenses_available=7,
         )
 
+        axis360.BIBLIOGRAPHIC_DATA.circulation = new_circulation
+
         e2, e_new, lp2, lp_new = api.update_book(
             axis360.BIBLIOGRAPHIC_DATA,
             new_circulation,
```

Comment:
Shouldn't update a constant in a test like this. It will persist across tests, potentially causing issues for tests run after it.

I'd suggest something like: 
```suggestion
        biblographic = copy.deepcopy(axis360.BIBLIOGRAPHIC_DATA)
        bibliographic.circulation = new_circulation

        e2, e_new, lp2, lp_new = api.update_book(
            biblographic,
            new_circulation,
```